Status/Debuffs/Debuff_Poison.gd

```diff
@@ -18,8 +18,8 @@ func init_status(in_caster: Entity, in_target: Entity) -> void:
 
 # @Override
 func on_turn_start() -> void:
-	var damage_data: DealDamageData = DealDamageData.new()
-	damage_data.damage = status_power
-	damage_data.caster = status_caster
+
+	var damage: float  = status_power
+	var caster: Entity = status_caster
```

Comment:
I don't think we need these local variables here, and can instead just use `status_power` and `status_caster` instead.

Reply:
I agree, can just directly use the variables